hw/ip/kmac/pre_dv/kmac_reduced_tb/README.md

```diff
@@ -0,0 +1,27 @@
+KMAC Reduced Verilator Testbench
+================================
+
+This directory contains a very basic, scratch Verilator testbench of the SHA3 core and the PRNG instantiated inside KMAC.
+The main use of this testbench is to help understanding how to operate and properly interface them, e.g., for evaluating security properties.
+
+How to build and run the testbench
+----------------------------------
+
+From the OpenTitan top level execute
+
+   ```sh
```

Comment:
Nit: I'm not sure the back-ticks need to be indented here

Reply:
You're right, it also looks weird in the source. Will change that.